src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins_zh_CN.properties

```diff
@@ -185,7 +185,7 @@ main.plugin.class=\u63D2\u4EF6\u7C7B
 
 main.plugin.module=\u63D2\u4EF6\u6A21\u5757
 
-main.plugin.category=\u7C7B\u522B 
+main.plugin.category=\u7C7B\u522B
```

Comment:
what's this?

Reply:
Removing the trailing space? A similar one is in the `ja` resource bundle.

Reply:
This happens as a combination of jcheck not enforcing eol space checks on .properties files, and my editor being set to always remove space at eol.